horace_core/sqw/@sqw/private/cut_accumulate_data_.m

```diff
@@ -168,3 +199,43 @@
     s(no_pix) = 0;
     e(no_pix) = 0;
 end
+
+
+function pci = init_pix_combine_info(nfiles, nbins)
+    % Create a pix_combine_info object to manage temporary files of pixels
+    wk_dir = get(parallel_config, 'working_directory');
+    gen_fpath = @(x) fullfile( ...
+        wk_dir, ['horace_subcut_', rand_digit_string(16), '.tmp'] ...
+    );
+    tmp_file_names = cellfun(gen_fpath, cell(1, nfiles), 'UniformOutput', false);
+    pci = pix_combine_info(tmp_file_names, nbins);
+end
+
+
+function str = rand_digit_string(n)
+    % Create string of n random digits
+    rand_ints = randi([0, 9], [1, n]);
```

Comment:
I can't actually see this combination of arguments in the randi docs - could you point me to it?

Reply:
```matlab
randi([imin, imax], sz)
```
`sz` can be a row vector of integers:
https://www.mathworks.com/help/matlab/ref/randi.html#buf2csg-sz
> sz — Size of each dimension (as a row vector)
> integer values

So the call generates a `[1, n]` array of random integers between 0 and 9 (inclusive).